.github/workflows/lint.yml

```diff
@@ -31,6 +31,5 @@ jobs:
       - name: Lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.60.3
-          working-directory: ${{ matrix.workdir }}
-          skip-pkg-cache: true
```

Comment:
`skip-pkg-cache` is remove in v5.0.0+.

https://github.com/marketplace/actions/golangci-lint#compatibility